lib/broadside/ecs/ecs_deploy.rb

```diff
@@ -100,67 +98,55 @@ def run
       end
     end
 
-    def status
-      super do
-        ips = EcsManager.get_running_instance_ips(@target.cluster, family)
-        info "\n---------------",
-          "\nDeployed task definition information:\n",
-          Rainbow(PP.pp(EcsManager.get_latest_task_definition(family), '')).blue,
-          "\nPrivate ips of instances running containers:\n",
-          Rainbow(ips.join(' ')).blue,
-          "\n\nssh command:\n#{Rainbow(gen_ssh_cmd(ips.first)).cyan}",
-          "\n---------------\n"
-      end
-    end
-
     def logtail
       super do
-        ip = get_running_instance_ip
+        ip = get_running_instance_ip!
```

Comment:
are you using the `!` just because it might exception out?  i guess rails kinda does that but i think the standard convention is to use `!` when you mutate something, and there's no mutation here.